commands/Serve.ts

```diff
@@ -57,29 +57,25 @@ export default class Serve extends BaseCommand {
   /**
    * Arguments to pass to the `encore` binary
    */
-  @flags.array({ description: 'CLI options to pass to the encore command line' })
-  public encoreArgs: string[] = []
+  @flags.array({ description: 'CLI options to pass to the assets bundler command line' })
+  public assetsBundlerArgs: string[] = []
```

Comment:
Forgot to mention that:
it's a breaking change. but i think we don't have to upgrade to a major version for that. what do you think?

Reply:
If it is a breaking change, we must majorly bump our version.

We can avoid that by keeping `encoreArgs` but deprecating it via JSDoc and a runtime warning.
This argument will be copied in `assetsBundlerArgs`.

Reply:
Yeah, good idea, gonna do that, thanks 👌